File: jooq/src/main/java/com/example/jooq/app/adapter/persistence/entity/Member.java
package com.example.jooq.app.adapter.persistence.entity;

import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Member {
    @Id
    @Column(name = "member_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    public Member(String name) {
        this.name = name;
    }
}

File: kotlin-multi-module/kotlin-multi-module/module-rds/src/main/kotlin/com/example/kotlinmultimodule/infra/domain/Member.kt
package com.example.kotlinmultimodule.infra.domain

import javax.persistence.*

@Entity
data class Member(
    @Id
    @Column(name = "member_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var name: String = ""
) : BaseEntity()
